server/pom.xml

```diff
@@ -129,13 +129,12 @@
       </exclusions>
     </dependency>
     <dependency>
-      <groupId>commons-collections</groupId>
-      <artifactId>commons-collections</artifactId>
-      <version>${commons-collections.version}</version>
+      <groupId>org.apache.commons</groupId>
```

Comment:
should only modify the `commons-collections` part, no need to touch the `commons-lang3` part

Reply:
Others is fine to me. But please check whether there are cascading dependencies on commons-collections 3. If there are, whether it is necessary to retain the definition of commons-collections 3 version in uniffle. @rickyma

Reply:
I just changed the order of the dependencies. `commons-lang3` remains the same as before. 

The UTs have all passed, so the casading dependencies should be OK.

Reply:
OK

Reply:
We can find commons-collections 3 in the dependency tree which is introduced by `hadoop-common`, but it is provided:
```
[INFO] +- org.apache.hadoop:hadoop-common:jar:2.8.5:provided
[INFO] |  +- commons-collections:commons-collections:jar:3.2.2:provided
```
@LuciferYang